src/Interface/Modules/Render/ViewScene.cc

```diff
@@ -1745,6 +1745,7 @@ void ViewSceneDialog::showEvent(QShowEvent* evt)
 
   if (pulledSavedVisibility_)
   {
+    ScopedWidgetSignalBlocker ssb(this);
```

Comment:
This is the one pertinent change, that prevents memory leakage from unnecessary re-execution.